activemodel/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+*   Numericality validation should skip `:greater_than`, `:greater_than_or_equal_to`, `:equal_to`,
```

Comment:
We should remove `:equal_to` from here as well.

> Well, I think you're right and I should remove it - since it does not compare numbers it won't crash in a way > described above. Also, it would cause a following confusing behavior:
> class Smth
>   include ActiveModel::Validations
>   validates :field_one, numericality: { allow_nil: true }
>   validates :field_two, numericality: { allow_nil: true, equal_to: :field_one }
> end
> 
> Smth.new(field_two: 1).valid? # => true, but why? these two are not equal

We should probably add test case for that in order to prevent any regression in the feature and describe the existing behavior

Reply:
Agreed, added one more test case